src/shipit_code_coverage_backend/shipit_code_coverage_backend/services/active_data.py

```diff
@@ -64,6 +61,12 @@ def base_query(filters=[], excludes=[]):
             {'term': {ActiveDataCoverage.FIELD_TEST_SUITE: 'awsy'}},
             {'term': {ActiveDataCoverage.FIELD_TEST_SUITE: 'talos'}},
 
+            # Ignore jsdcov builds
+            {'term': {ActiveDataCoverage.FIELD_BUILD_TYPE: 'jsdcov'}},
+
+            # Ignore per-test suites by disabling debug-test runs
+            {'prefix': {ActiveDataCoverage.FIELD_RUN_NAME: 'test-linux64-ccov/debug-test-coverage-'}},
```

Comment:
It can also be windows10 or mac, we should actually ignore anything *-test-coverage-*

Reply:
Done with wildcard